# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Position Controllers ---------------------------------------
joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint: Joint_1
  pid: {p: 30.0, i: 0, d: 2.0}
joint2_position_controller:
  type: effort_controllers/JointPositionController
  joint: Joint_2
  pid: {p: 30.0, i: 0, d: 2.0}
joint3_position_controller:
  type: effort_controllers/JointPositionController
  joint: Joint_3
  pid: {p: 30.0, i: 0, d: 2.0}
joint4_position_controller:
  type: velocity_controllers/JointPositionController
  joint: Joint_4
  pid: {p: 1.0, i: 0.0, d: 1.0}
